File: src/exponential_decay.py
# https://www.boot.dev/lessons/3ae4708e-0a55-4bae-8011-6de75bce05ab
def decayed_followers(intl_followers: int,
                      fraction_lost: float,
                      days: int) -> int:
    """remaining_total = quantity * ( retention_rate ^ time )"""
    remaining_followers = intl_followers * ((1 - fraction_lost) 

File: src/ExponentialDecay.java
class ExponentialDecay{
// remaining_total = quantity * ( retention_rate ^ time )
    public static void main(String[] args){
        if (args.length < 3){
            IO.println("Please provide correct arguments.");
            IO.println("initialFollowers, fractionLost, days");
            return;
        }
        int initialFollowers = Integer.parseInt(args[0]);
        double fractionLost = Double.parseDouble(args[1]);
        int days = Integer.parseInt(args[2]);
        double remaining = calculateRemaining(initialFollowers,
                                              fractionLost,
                                              days);
        IO.println((int) remaining);
    }

    public static double calculateRemaining(int initialFollowers,
                                            double fractionLost,
                                            int days){
    double remaining = initialFollowers * Math.pow(1 - fractionLost, (double) days);
    return remaining;
    }
}

